src/constants.ts

```diff
@@ -1,24 +1,13 @@
-export const operators = {
-  and: 'and',
-  or: 'or',
-
-  eq: '==',
-  gt: '>',
-  gte: '>=',
-  lt: '<',
-  lte: '<=',
-  ne: '!=',
-
-  add: '+',
-  subtract: '-',
-  multiply: '*',
-  divide: '/',
-  pow: '^',
-  mod: '%',
-
-  in: 'in',
-  'not in': 'not in'
-}
+// operator precedence from lowest to highest
+export const operators = [
```

Comment:
By the way, did you arrange these orders the same as how JS works?

Reply:
O that is a good point. There where some differences, I've now aligned the precedence with that of JS, see 60a66932387e3b974f5bdd59ee3687124c919811. I'll also invert the order of the list to make it more natural to read.